### ISSUE TITLE
bug: race condition on QR scans leads to process destroying error

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1552]
Based on the error logs, the error only occurs when a QR code scan is successful. The error should be handled and not thrown to the main process.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.10.6](https://github.com/open-wa/wa-automate-nodejs/compare/3.10.5...3.10.6) (2021-04-03)

- build(deps-dev): bump release-it from 14.5.0 to 14.5.1 [`#1547`](https://github.com/open-wa/wa-automate-nodejs/pull/1547)
- build(deps): bump swagger-ui-dist from 3.45.1 to 3.46.0 [`#1549`](https://github.com/open-wa/wa-automate-nodejs/pull/1549)
- build(deps): bump @types/sharp from 0.27.2 to 0.27.3 [`#1550`](https://github.com/open-wa/wa-automate-nodejs/pull/1550)
- build(deps-dev): bump @typescript-eslint/parser from 4.19.0 to 4.20.0 [`#1541`](https://github.com/open-wa/wa-automate-nodejs/pull/1541)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.19.0 to 4.20.0 [`#1540`](https://github.com/open-wa/wa-automate-nodejs/pull/1540)
- build(deps): bump p-queue from 6.6.2 to 7.0.0 [`#1545`](https://github.com/open-wa/wa-automate-nodejs/pull/1545)
- build(deps-dev): bump @types/marked from 2.0.0 to 2.0.1 [`#1544`](https://github.com/open-wa/wa-automate-nodejs/pull/1544)
- 🐛  attempt to fix #1552 [`#1552`](https://github.com/open-wa/wa-automate-nodejs/issues/1552)
- 🥚  update patches, fixes #1539 [`#1539`](https://github.com/open-wa/wa-automate-nodejs/issues/1539)
- Release 3.10.6 [`c0cdba0`](https://github.com/open-wa/wa-automate-nodejs/commit/c0cdba0132fae4995e388c8394712dfa5609bd85)
- 🐛  prevent `undeinfed` `phone` from causing error on `loaded` [`3892f48`](https://github.com/open-wa/wa-automate-nodejs/commit/3892f48d9e82a0b181a2f037507f67a7c2bec2bd)
- prevent error shown in #1538 [`597d68a`](https://github.com/open-wa/wa-automate-nodejs/commit/597d68a37bdaa7da22e74023b3d7ff8df52ee2dc)
- 🥚  patch update [`c565d90`](https://github.com/open-wa/wa-automate-nodejs/commit/c565d909ee7072eb7887e56a5870b74d589bef4b)
- 🥚  patches update [`b0a1f04`](https://github.com/open-wa/wa-automate-nodejs/commit/b0a1f04842129149450e45cbd117b858b9011e28)
